### ISSUE TITLE
Return the image of the article

### ISSUE THREAD

**Issue by tikiki** [WikiJourney/wikijourney_website#24]
It would be a good idea to include a link to the main image of the article in the API response.

For example, give something like this: 
```
{"name":"Lyon",
"image_url":"https://upload.wikimedia.org/wikipedia/commons/thumb/9/90/Arrondissements_de_Lyon.svg/885px-Arrondissements_de_Lyon.svg.png"
}
```
Well, maybe not the best image example, but still...

**Comment by sylvainar** [comment]
I'm currently on it, thanks for asking ! :smile:

**Comment by sylvainar** [comment]
Implemented on commit a60db285702e294611bd03ebe5e74fc4cb0c219e !
